filebeat/tests/test.sh

```diff
@@ -1,32 +1,31 @@
 #!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
 
-TESTDIR="$(dirname "${0}")"
-PLANDIR="$(dirname "${TESTDIR}")"
-SKIPBUILD=${SKIPBUILD:-0}
+set -euo pipefail
 
-hab pkg install --binlink core/bats
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
 
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+hab pkg install core/bats --binlink
 hab pkg install core/busybox-static
 hab pkg binlink core/busybox-static ps
 hab pkg binlink core/busybox-static wc
+hab pkg install "${TEST_PKG_IDENT}"
 
-source "${PLANDIR}/plan.sh"
-
-if [ "${SKIPBUILD}" -eq 0 ]; then
-  # Unload the service if its already loaded.
-  hab svc unload "${HAB_ORIGIN}/${pkg_name}"
+hab svc load "${TEST_PKG_IDENT}"
```

Comment:
The test setup fails on my workstation (and I believe will also fail on Buildkite) because the supervisor is not running.

If I add 

```
hab sup term
hab sup run &
sleep 10
```

(and also a `hab sup term` at the end of this script) everything passes.

Reply:
What if you *just* add the sleep 10 before `svc load`?

Reply:
Ah, you mean the supervisor should be running?

That still doesn't work. Is this a characteristic of `hab studio run` ?

If I enter a studio manually and carry out the steps of `test.sh` then the sup is running.

Reply:
Oh right.. Thanks @james-stocks